calico-cloud/operations/disconnect.mdx

```diff
@@ -23,6 +23,14 @@ To remove policies from tiers, you have these options:
 - Manually move policies out of tiers prior to running the script
 - Let the script remove _ALL_ Calico policies by specifying the `--remove-all-calico-policy` flag
 
+:::note
+
+If you are keeping policies in the default tier, especially GlobalNetworkPolicies, and you have default
```

Comment:
If you want to keep policies in the default tier (especially GlobalNetworkPolicies), and you have default
deny policies, update the policies so they do not apply to {{prodname}} namespaces and Calico components, or there may be issues with the script.

Reply:
I was purposefully trying to say both Calico Cloud or Calico namespaces. Because the downgrade switches to an OSS configuration so the policies need to "not apply" to both namespaces used in Calico Cloud and in an OSS Calico configuration.

Reply:
I think above wording is now clear. If this isn't better, I trust your instincts over mine.

To successfully downgrade to an open-source Calico configuration, policies cannot apply to either {{prodname}} or open-source Calico namespaces. If you want to keep policies in the default tier (especially GlobalNetworkPolicies), and you have default deny policies, update the policies namespaces accordingly.

Reply:
How about:

To successfully downgrade to an open-source Calico configuration, policies must allow necessary traffic to and from {{prodname}} and open-source Calico namespaces. If you keep policies in the default tier (especially GlobalNetworkPolicies), and you have default deny policies, update or add policies accordingly to allow this necessary traffic.